docs/API.md

```diff
@@ -1376,6 +1399,10 @@ newer revisions. The fully qualified ids of those charms will be returned in an
 ordered list from newest to oldest revision. Note that the current revision
 will be included in the list as it is also an available revision.
 
+If a charm supports multiple series, and the series is not specified in the
```

Comment:
The endpoints use router.ResolvedURL which will contain an empty series if none is specified in the request. So the endpoint code does know doesn't it?

I do think that the example I gave where the series are listed out is valid. I anticipated that core would use it when upgrading to see what maximum revision of a charm could be used which supports are series currently running workloads for that charm. So if the user types "juju upgrade-charm", juju would make a revision-info call with just the charm name to get the info it needs.